packages/library/build.gradle.kts

```diff
@@ -62,6 +73,10 @@ kotlin {
 //    }
 }
 
+project.extensions.configure(kotlinx.atomicfu.plugin.gradle.AtomicFUPluginExtension::class) {
```

Comment:
There was a reason for not applying it on JVM, correct? Maybe add a note about it?